cpanfile

```diff
@@ -3,4 +3,5 @@ requires 'Locale::gettext', '1.0';
 on test => sub {
   requires 'Test2::V0', '0.000126';
   requires 'Test2::Harness', '0.001099';
+  requires 'Test::More', '0.88';
```

Comment:
I think that this test will in the future be trivial to port to Test2::V0 but no reason to block on this at the moment.